src/GraphWebhooks-Core/Models/KeyVaultOptions.cs

```diff
@@ -9,5 +9,6 @@ public class KeyVaultOptions
         public string ClientId { get; set; }
         public string ClientSecret { get; set; }
         public string CertificateUrl { get; set; }
+        public string KeyVaultUri { get; set; }
```

Comment:
Do we need both the KeyVault URL and the certificate URL?
Couldn't we:
- deduce the keyvault URL from the certificate URL -OR-
- ask for the certificate path 

?

Reply:
Deducing the keyvault URL from the certificate URL